src/style.css

```diff
@@ -1 +1,61 @@
-/* add styles here */
+
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .stars {
+    display: flex;
+
+    .stars__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background: url(images/star.svg) no-repeat center;
+    }
+  }
+
+  .stars--0 {
+    .stars__star:nth-child(-n) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+
+  .stars--1 {
+    .stars__star:nth-child(-n + 1) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+
+  .stars--2 {
+    .stars__star:nth-child(-n + 2) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+
+  .stars--3 {
+    .stars__star:nth-child(-n + 3) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+
+  .stars--4 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+
+  .stars--5 {
+    .stars__star:nth-child(n) {
+      background-image: url(images/star-active.svg);
+    }
+  }
+}
```

Comment:
You can simplify it
To do something like this
<img width="447" alt="image" src="https://github.com/mate-academy/layout_stars/assets/86789161/d6b729c0-3727-4ed9-a848-fd59249e557c">